connectors/src/connectors/zendesk/lib/zendesk_api.ts

```diff
@@ -131,8 +131,13 @@ async function fetchFromZendeskWithRetries({
     }
   }
   const text = await rawResponse.text();
-  const response = JSON.parse(text);
-
+  let response;
```

Comment:
Why getting it as text to then parse it as json? 
You could just do const response = rawResponse.json() in the try catch directly?